### PR TITLE
Mgmt func history

### PR DESCRIPTION
**Pull request by zacharyhamm**
Shows a history of all management function executions for the change set
in the history tab, and allows you to see the latest execution logs via
the component's management function panel.

### CODE REVIEW

File: app/web/src/components/ChangesPanelHistory.vue [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <template>
-  <ScrollArea ref="mainDivRef" @click="deselectOnClickEmptySpace">
+  <div
+    v-if="featureFlagsStore.MANAGEMENT_FUNCTIONS"
```
Comment (zacharyhamm) [inline app/web/src/components/ChangesPanelHistory.vue:3]: @wendybujalski I'll pull this down to test the flag in a second. But will this hide the *entire* history if you don't have the flag on? We just want to hide the management func history

Comment (wendybujalski) [reply app/web/src/components/ChangesPanelHistory.vue:3]: Nah, I just separated the two options for flag on/off to make it simpler.

File: app/web/src/components/ChangesPanelHistory.vue [hunk 2]
```diff
@@ -1,69 +1,143 @@
...
     />
-  </ScrollArea>
-  <EmptyStateCard
-    v-else
-    iconName="actions"
-    primaryText="No Actions Have Been Taken"
-    secondaryText="There is no action history to display for this change set."
-  />
+  </div>
+  <div v-else ref="actionDivRef" class="py-2xs w-full h-full">
```
Comment (wendybujalski) [inline app/web/src/components/ChangesPanelHistory.vue:77]: @zacharyhamm Look here!

Comment (zacharyhamm) [reply app/web/src/components/ChangesPanelHistory.vue:77]: ahh! i see! very clever

Comment (zacharyhamm) [reply app/web/src/components/ChangesPanelHistory.vue:77]: that was hard to parse in the diff but it's clear from the full file.

File: lib/veritech-core/src/lib.rs [hunk 3]
```diff
@@ -222,7 +222,7 @@ impl VeritechRequest {
             VeritechRequest::KillExecution(kill_execution_request) => {
                 &kill_execution_request.execution_id
             }
-            VeritechRequest::Mangement(management_request) => &management_request.execution_id,
+            VeritechRequest::Management(management_request) => &management_request.execution_id,
```
Comment (vbustamante) [inline lib/veritech-core/src/lib.rs:225]: 💯

**Review by jobelenus — APPROVED** [review]
